res/css/views/messages/_DecryptionFailureBody.pcss

```diff
@@ -10,3 +10,24 @@ Please see LICENSE files in the repository root for full details.
     color: $secondary-content;
     font-style: italic;
 }
+
+/* Formatting for the "Verified identity has changed" error */
+.mx_DecryptionFailureVerifiedIdentityChanged > span {
+    /* Show it in red */
+    color: $e2e-warning-color;
+    /* TODO: background colour? */
+
+    /* With a red border */
+    border: 1px solid $e2e-warning-color;
+    border-radius: $font-16px;
```

Comment:
Compound spacing token!

Reply:
ditto

Reply:
I this one needs to scale with the font size, otherwise we get a different shape at large font sizes:

![image](https://github.com/user-attachments/assets/0c0e0d31-baff-4cba-bab0-fb99f74a932e)